### PR TITLE
Feature/enable scl python

### PR DESCRIPTION
**Pull request by antevens**

### CODE REVIEW

**Comment by hrpatel** [comment]
@antevens Where is `enable_scl_python`?

**Comment by antevens** [comment]
Missing the SCL code, will address tomorrow.

File: shtdlib.sh [hunk 1]
```diff
@@ -793,6 +793,18 @@ function init_nss_wrapper {
     export NSS_WRAPPER_HOSTS="${tmp_hosts_file}"
 }
 
+function enable_scl_python {
+    assert [ "${os_name}" = "redhat" ]
+    shopt_decorator_option_name='nounset'
+    shopt_decorator_option_value='false'
+    python_version="${1:-36}"
```
Comment (hrpatel) [inline shtdlib.sh]: Default should be `3.6`

File: shtdlib.sh [hunk 2]
```diff
@@ -793,6 +793,18 @@ function init_nss_wrapper {
     export NSS_WRAPPER_HOSTS="${tmp_hosts_file}"
 }
 
+function enable_scl_python {
```
Comment (hrpatel) [inline shtdlib.sh]: Short function summary along with whats happening with trim

File: shtdlib.sh (hunk 1 above)
Comment (antevens) [reply shtdlib.sh]: Are you sure, I thought 36 was used in the path, unless we want to have a routine that strips dots and minor versions?

File: shtdlib.sh (hunk 2 above)
Comment (antevens) [reply shtdlib.sh]: Done

File: shtdlib.sh [hunk 3]
```diff
@@ -793,17 +798,20 @@ function init_nss_wrapper {
...
 }
 
-# Enable Software Collections, allows multiple versions of the same RPMs to be
+# Enable a Python Software Collection, SCL allows multiple versions of the same RPMs to be
 # installed at the same time.
 function enable_scl_python {
     assert [ "${os_name}" = "redhat" ]
     shopt_decorator_option_name='nounset'
     shopt_decorator_option_value='false'
     python_version="${1}"
```
Comment (hrpatel) [inline shtdlib.sh]: Set a default, `3.6`, or assert that `${1}` is non-empty

Comment (antevens) [reply shtdlib.sh]: Oks, I felt it was a bit redundant but you're right, people should not be surprised when running in non-strict mode

**Pull request merged**